core/src/main/resources/error/error-classes.json

```diff
@@ -668,6 +668,24 @@
       }
     }
   },
+  "LOCATION_ALREADY_EXISTS" : {
+    "message" : [
+      "Cannot create the location <location> because it already exists.",
+      "Choose a different path or remove the existing location."
+    ],
+    "subClass" : {
```

Comment:
Is this level of differentiation necessary?
Aside the sub-error class partially repeats what the general test says.
To make this message useful I think we should explain how/that the the  table's directory name (?) is derived from the table name?

Reply:
> I think we should explain how/that the the table's directory name (?) is derived from the table name?

ok. Added a few words. Don't think we should put too much details about impl. It is more important to say how to fix the issue.

Reply:
Still not a fan of the subclasses... But see new comment above.